app/navigation/Navigator.scala

```diff
@@ -166,7 +166,7 @@ class Navigator @Inject() () {
           } else {
             routes.CommodityCodeController.onPageLoad(CheckMode)
           }
-        case false => routes.CyaCreateRecordController.onPageLoad
```

Comment:
this was a navigation bug that mean that the app was trying to create a valid GoodsRecord when it should have instead been redirected to the CommodityCodePage to recheck the code